packages/reports/src/Components/Form/CompositeCalendar/composite-calendar-mobile.tsx

```diff
@@ -42,7 +42,7 @@ export const RadioButton = ({ id, className, selected_value, value, label, onCha
 };
 const CUSTOM_KEY = 'custom';
 
-type TInputDateRange = {
+export type TInputDateRange = {
```

Comment:
please move reusable types to the `packages/reports/src/Types/common-prop.type.ts`